### PR TITLE
Add midiStroke

### PR DESCRIPTION
**Pull request by leafac**
#### Adding a new cask
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/midi-stroke.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+cask 'midi-stroke' do
```
Comment (vitorgalvao) [inline Casks/midi-stroke.rb]: You did **not** follow the token reference. It makes clear this needs to be `midistroke`.

File: Casks/midi-stroke.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  sha256 'b6c3215df9baffa11ab2afe6e33e550697ea09c01e82d3f51aab7eefabaf65f4'
+
+  url 'http://www.charlie-roberts.com/midiStroke/midiStroke.app.zip'
+  name 'midiStroke'
+  homepage 'http://www.charlie-roberts.com/midiStroke/'
+  license :mit
+
+  app 'midiStroke.app'
+
+  zap delete: [
```
Comment (vitorgalvao) [inline Casks/midi-stroke.rb]: Change to `zap delete: '~/Library/Application Support/midiStroke'` (one line).

File: Casks/midi-stroke.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
+cask 'midi-stroke' do
+  version '1.4'
```
Comment (vitorgalvao) [inline Casks/midi-stroke.rb]: Seeing as `url` doesn’t point to a specific version, this cask should use `version :latest` and `sha256 :no_check`. [See an example](https://github.com/caskroom/homebrew-cask/blob/f25b6babcd398abf48e33af3d887b2d00de1d661/Casks/kibako.rb).

**Comment by leafac** [comment]
@vitorgalvao, first, thank you for reviewing my pull request so quickly. I performed the changes you suggested.

**Comment by vitorgalvao** [comment]
Thank you. As soon as Travis passes, we can merge.

**Comment by leafac** [comment]
> The Travis CI build passed

:shipit: 

:heart:

**Pull request merged**